### PR TITLE
Add com.daidouji.oneko

### PR DESCRIPTION
**Pull request by jcleng**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: org.daidouji.oneko.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+    "app-id": "org.daidouji.oneko",
```
Comment (hfiguiere) [inline org.daidouji.oneko.json:2]: `daidouji.org`  doesn't exist.

File: org.daidouji.oneko.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "/app/bin/oneko",
```
Comment (hfiguiere) [inline org.daidouji.oneko.json:6]:
```suggestion
    "command": "oneko",
```

File: org.daidouji.oneko.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
```
Comment (hfiguiere) [inline org.daidouji.oneko.json:10]:
```suggestion
```

File: org.daidouji.oneko.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        "--device=dri",
+        "--socket=x11",
+        "--filesystem=home"
```
Comment (hfiguiere) [inline org.daidouji.oneko.json:13]: why do a game need the whole home access?

File: org.daidouji.oneko.json [hunk 5]
```diff
@@ -0,0 +1,52 @@
+{
+    "app-id": "org.daidouji.oneko",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "/app/bin/oneko",
+    "finish-args": [
+        "--share=network",
```
Comment (hfiguiere) [inline org.daidouji.oneko.json:8]: why is network access needed.

File: org.daidouji.oneko.json [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+                {
+                    "type": "dir",
+                    "path": "./",
+                    "skip": ".flatpak-builder"
+                }
+            ]
+        },
+        {
+            "name": "oneko",
+            "buildsystem": "cmake",
```
Comment (hfiguiere) [inline org.daidouji.oneko.json:36]: does it work with `cmake-ninja`. it is prefered if it does.

File: org.daidouji.oneko.metainfo.xml [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+  <url type="homepage">http://www.daidouji.com/oneko/</url>
+  <screenshots>
+    <screenshot type="default">https://raw.githubusercontent.com/jcleng/org.daidouji.oneko/org.daidouji.oneko/screenshots/1.jpg</screenshot>
+    <screenshot >https://raw.githubusercontent.com/jcleng/org.daidouji.oneko/org.daidouji.oneko/screenshots/2.jpg</screenshot>
+  </screenshots>
+  <releases>
+    <release version="1.2.0" date="2023-09-05"/>
+  </releases>
+  <updatecontact>j.c.leng@foxmail.com</updatecontact>
+</application>
```
Comment (hfiguiere) [inline org.daidouji.oneko.metainfo.xml:26]: missing required `content_rating`

File: org.daidouji.oneko.json (hunk 1 above)
Comment (hfiguiere) [reply org.daidouji.oneko.json:2]: elsewhere `.com` is used.

File: org.daidouji.oneko.json [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=x11",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "install desktop",
```
Comment (hfiguiere) [inline org.daidouji.oneko.json:17]: this should be part if the "main" module below. there is not point splitting it.

**Comment by hfiguiere** [comment]
bot, build com.daidouji.oneko

File: com.daidouji.oneko.json [hunk 9]
```diff
@@ -0,0 +1,48 @@
+{
+    "app-id": "com.daidouji.oneko",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "oneko",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11"
```
[the remark this answers is not in the document]
Comment (jcleng) [reply com.daidouji.oneko.json:9]: yes, without

**Comment by jcleng** [comment]
bot, build com.daidouji.oneko

**Pull request closed without merging**